pkg/liqo-controller-manager/foreign-cluster-operator/foreign-cluster-controller.go

```diff
@@ -106,6 +98,8 @@ type ForeignClusterReconciler struct {
 
 	InsecureTransport *http.Transport
 	SecureTransport   *http.Transport
+
+	ForeignClusters sync.Map
```

Comment:
Would suggest to add a comment about the content of the map (i.e., key and value), for future reference.